### PR TITLE
Documentation for HFP2.0

### PR DESCRIPTION
**Pull request by mjaakko**

### CODE REVIEW

File: src/pages/en/developers/apis/4-realtime-api/vehicle-positions-2/index.md [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+| `route`   | String                | `da`, `dout`, `ba`, `bout`           | ID of the route the vehicle is currently running on. Matches `route_id` in the topic.
+| `occu`    | Integer               | `da`, `dout`, `ba`, `bout`           | Integer describing passenger occupancy level of the vehicle. Valid values are on interval `[0, 100]`. However, currently only values used are `0` *(= vehicle has space and is accepting passengers)* and `100` *(= vehicle is full and might not accept passengers)*
+| `ttarr`   | String                | `vp`, `da`, `dout`, `ba`, `bout`, `vja`, `vjout` | UTC timestamp of scheduled arrival time to the stop
+| `ttdep`   | String                | `vp`, `da`, `dout`, `ba`, `bout`, `vja`, `vjout` | UTC timestamp of scheduled departure time from the stop
+| `dr-type` | Integer               | Other than `da`, `dout`, `ba`, `bout`, `vja`, `vjout` | Type of the driver, either `0` or `1`. <ul><li>`0` = normal driver</li><li>`1` = service technician</li></ul>
```
Comment (kenkus-futurice) [inline src/pages/en/developers/apis/4-realtime-api/vehicle-positions-2/index.md]: Is this the correct way? According to docs:
"Driver Sign-in type (1 = driver, 0 = service technician)"

File: src/pages/en/developers/apis/4-realtime-api/vehicle-positions-2/index.md [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+| `tlp-requestid` | Integer              | Other than `tlr`, `tla`                                  | Traffic light priority request ID. Valid values are on interval `[0, 100]`.
+| `tlp-requesttype` | String             | Other than `tlr`                                         | Traffic light priority request type, either `NORMAL`, `DOOR_CLOSE`, `DOOR_OPEN` or `ADVANCE`.
+| `tlp-prioritylevel`  | String          | Other than `tlr`                                         | Priority level of a traffic light priority request. Either `normal`, `high` or `norequest`.
+| `tlp-reason`    | String               | Other than `tlr`                                         | Reason for *not* sending a traffic light priority request. Either `GLOBAL`, `AHEAD`, `LINE` or `PRIOEXEP`.
+| `tlp-att-seq`   | Integer              | Other than `tlr`                                         | Traffic light priority request attempt sequence number-
```
Comment (kenkus-futurice) [inline src/pages/en/developers/apis/4-realtime-api/vehicle-positions-2/index.md]: Typo `-` at the end?

File: src/pages/en/developers/apis/4-realtime-api/index.md [hunk 3]
```diff
@@ -12,6 +12,7 @@ At first you should read the real-time API [getting started guide](./1-getting-s
 |----------------------------------------------------|------------------------|
 | [Service alerts](./service-alerts/)                | Provides disruption information from HSL area in [GTFS-RT](https://developers.google.com/transit/gtfs-realtime/) format
 | [Trip updates](./trip-updates/)                    | Provides predictions in [GTFS-RT](https://developers.google.com/transit/gtfs-realtime/) format to arrival and departures times for stops along the route of a trip.
-| [High frequency positioning](./vehicle-positions/) | Provides vehicle positions from HSL area in JSON format over MQTT
+| [High frequency positioning (v2.0)](./vehicle-positions-2/) | Provides vehicle positions from HSL area in JSON format over MQTT
```
Comment (kenkus-futurice) [inline src/pages/en/developers/apis/4-realtime-api/index.md]: Should we mention also events?

File: src/pages/en/developers/apis/4-realtime-api/vehicle-positions/index.md [hunk 4]
```diff
@@ -12,6 +13,9 @@ replit:
       height: 800px
 ---
 
+[[alert-warning]]
+| **Note:** HFP 1.0 is deprecated and vehicles will stop sending HFP 1.0 messages in the future. Please migrate to [HFP 2.0](./vehicle-positions-2) if you're using HFP 1.0.
```
[the remark this answers is not in the document]
Comment (mjaakko) [reply src/pages/en/developers/apis/4-realtime-api/vehicle-positions/index.md]: Fixed

File: src/pages/en/developers/apis/4-realtime-api/vehicle-positions-2/index.md (hunk 1 above)
Comment (mjaakko) [reply src/pages/en/developers/apis/4-realtime-api/vehicle-positions-2/index.md]: Fixed

File: src/pages/en/developers/apis/4-realtime-api/index.md (hunk 3 above)
Comment (mjaakko) [reply src/pages/en/developers/apis/4-realtime-api/index.md]: Yes, I added mention of events.

**Pull request merged**